DQM/Phase2OuterTracker/plugins/OuterTrackerMonitorTTTrack.cc

```diff
@@ -0,0 +1,365 @@
+{
+  /*
+  /// Track Trigger Tracks
+  edm::Handle< std::vector< TTTrack< Ref_Phase2TrackerDigi_ > > > Phase2TrackerDigiTTTrackHandle;
+  iEvent.getByToken( tagTTTracksToken_, Phase2TrackerDigiTTTrackHandle );
+  
+  unsigned int numHQTracks = 0;
+  unsigned int numLQTracks = 0;
+  unsigned int numTracks = 0; 
+  
+  /// Go on only if there are TTTracks from Phase2TrackerDigis
+  if ( Phase2TrackerDigiTTTrackHandle->size() > 0 )
+  {
+    /// Loop over TTTracks
+    unsigned int tkCnt = 0;
+    std::vector< TTTrack< Ref_Phase2TrackerDigi_ > >::const_iterator iterTTTrack;
+    for ( iterTTTrack = Phase2TrackerDigiTTTrackHandle->begin();iterTTTrack != Phase2TrackerDigiTTTrackHandle->end();++iterTTTrack )
+    {
+      /// Make the pointer
+      edm::Ptr< TTTrack< Ref_Phase2TrackerDigi_ > > tempTrackPtr( Phase2TrackerDigiTTTrackHandle, tkCnt++ );
+      numTracks++;
+      
+      unsigned int nStubs = tempTrackPtr->getStubRefs().size();
+      
+      double trackPt = tempTrackPtr->getMomentum().perp();
+      double trackPhi = tempTrackPtr->getMomentum().phi();
+      double trackEta = tempTrackPtr->getMomentum().eta();
+      double trackVtxZ0 = tempTrackPtr->getPOCA().z();
+      double trackChi2 = tempTrackPtr->getChi2();
+      double trackChi2R = tempTrackPtr->getChi2Red();
+      
+      Track_NStubs->Fill(nStubs);
+      
+      if ( nStubs >= HQDelim_ )
+      {
+        numHQTracks++;
+        
+        Track_HQ_Pt->Fill( trackPt );
+        Track_HQ_Eta->Fill( trackEta );
+        Track_HQ_Phi->Fill( trackPhi );
+        Track_HQ_VtxZ0->Fill( trackVtxZ0 );
+        Track_HQ_Chi2->Fill( trackChi2 );
+        Track_HQ_Chi2Red->Fill( trackChi2R );
+        
+        Track_HQ_Chi2_NStubs->Fill( nStubs, trackChi2 );
+        Track_HQ_Chi2Red_NStubs->Fill( nStubs, trackChi2R );
+      }
+      else
+      {
+        numLQTracks++;
+        
+        Track_LQ_Pt->Fill( trackPt );
+        Track_LQ_Eta->Fill( trackEta );
+        Track_LQ_Phi->Fill( trackPhi );
+        Track_LQ_VtxZ0->Fill( trackVtxZ0 );
+        Track_LQ_Chi2->Fill( trackChi2 );
+        Track_LQ_Chi2Red->Fill( trackChi2R );
+        
+        Track_LQ_Chi2_NStubs->Fill( nStubs, trackChi2 );
+        Track_LQ_Chi2Red_NStubs->Fill( nStubs, trackChi2R );
+      }
+    } // End of loop over TTTracks
+    
+  } // End TTTracks from pixeldigis 
+
+  Track_N->Fill(numTracks); 
+  Track_HQ_N->Fill( numHQTracks );
+  Track_LQ_N->Fill( numLQTracks );
+  */
+} // end of method
+
+
+// ------------ method called once each job just before starting event loop  ------------
+void
+OuterTrackerMonitorTTTrack::beginRun(const edm::Run& run, const edm::EventSetup& es)
+{
+  SiStripFolderOrganizer folder_organizer;
+  folder_organizer.setSiStripFolderName(topFolderName_);
+  folder_organizer.setSiStripFolder();	
+	std::string HistoName;
+  
+  dqmStore_->setCurrentFolder(topFolderName_+"/Tracks/");
```

Comment:
I'm unclear why there is a need for two dqmStores here (one inside of SiStripFolderOrganizer). Perhaps folder_organizer is not really used?